### DISCUSSION TITLE
Is there a way to only enable patch updates for Dockerfiles?

### DISCUSSION THREAD

**Discussion by ep-linden** [renovatebot/renovate#15861] [Request Help]
I don't want Renovate Bot to make update suggestions for minor patches.

There was an approach I did with `enabled: false`:

```
{
  "separateMinorPatch": true,
  "packageRules": [
    {
      "matchDatasources": ["docker"],
      "matchUpdateTypes": ["minor"],
      "enabled": false
    }
  ]
}
```

This works, but I don't want to set `enabled: false` because our Renovate bot is set to force `enabled: true`.

I'm just wondering if there are alternatives that do not use `enabled: false`.

**Comment by rarkins** [comment]
You could set `dependencyDashboardApproval` to `true` instead

**Comment by viceice** [comment]
you also need to enable separate minor and patch updates, otherwise patch updates are also disabled

**Reply by ChristianCiach** [reply]
I thought this comment would be the solution for why Renovate doesn't provide patch-updates for my release-branches:

```json
{
    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
    "extends": [
        "config:recommended"
    ],
    "assigneesFromCodeOwners": true,
    "semanticCommits": "disabled",
    "baseBranches": [
        "main",
        "/^release\\/.*/"
    ],
    "packageRules": [
        {
            "matchBaseBranches": [
                "/^release\\/.*/"
            ],
            "matchUpdateTypes": [
                "major",
                "minor"
            ],
            "separateMinorPatch": true,
            "enabled": false
        }
    ]
}
```

But unfortunately:

```
 WARN: Configuration Warning (repository=shared/code/platform/spring-dependencies)
       "configError": {
         "validationSource": "renovate.json",
         "validationError": "The renovate configuration file contains some invalid settings",
         "validationMessage": "packageRules[1]: packageRules cannot combine both matchUpdateTypes and separateMinorPatch. Rule: {\"matchBaseBranches\":[\"/^release\\\\/.*/\"],\"matchUpdateTypes\":[\"major\",\"minor\"],\"separateMinorPatch\":true,\"enabled
\":false}",
         "message": "config-validation",

```

**Reply by ChristianCiach** [reply]
I had to move `"separateMinorPatch": true` one level up, outside of the `packageRules`. Weird. I think there should be an easier way to restrict Renovate to updates of type "patch", but that's probably easier said than done.

**Reply by viceice** [reply]
you can use it in package rules, but not combine with match update types

**Reply by RPCMoritz** [reply]
What would a working config look like, that still affects only a single repository (i.e. fulfills the question asked in the title?)
